netlogo-core/src/test/fileformat/NLogoFormatTests.scala

```diff
@@ -111,6 +111,42 @@ class NLogoFormatConversionTest extends FunSuite with ConversionHelper {
       assert(rereadModel.code.contains("extensions [vid]"))
     }
 
+    test("performs tests for sectionToSource conversion conversion between files") {
```

Comment:
The string param here gives the test its name, so this test will be called `performs tests for...`, but we already know this will perform tests because it's a test.  A better name should be more of the form `NOUN SHOULD VERB WHEN CONDITION`.